x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_charts/risks_table.tsx

```diff
@@ -81,10 +42,11 @@ export const RisksTable = ({
   onCellClick,
   onViewAllClick,
 }: RisksTableProps) => {
-  const columns = useMemo(
+  const columns: Array<EuiBasicTableColumn<GroupedFindingsEvaluation>> = useMemo(
     () => [
       {
         field: 'name',
+        truncateText: true,
         name: TEXT.CIS_SECTION,
         render: (name: GroupedFindingsEvaluation['name']) => (
           <EuiLink onClick={() => onCellClick(name)}>{name}</EuiLink>
```

Comment:
i think because we use a custom `render`, we might need to add `className='eui-textTruncate'` to `EuiLink` to get the `text-elipsis` effect.

Reply:
👑👑👑👑👑👑👑👑👑